helix-lsp/src/lib.rs

```diff
@@ -614,6 +614,7 @@ pub enum Notification {
     ShowMessage(lsp::ShowMessageParams),
     LogMessage(lsp::LogMessageParams),
     ProgressMessage(lsp::ProgressParams),
+    WorkspaceDiagnosticRefresh,
```

Comment:
It looks like this is actually a request rather than a notification in the spec, so this should live under `MethodCall`. In `application.rs` the handler for this branch should move down to where we handle `Call::MethodCall`s and return `Ok(serde_json::Value::Null)` I believe

Reply:
You are right. I have moved it to `MethodCall`